Readability.js

```diff
@@ -469,9 +476,9 @@ Readability.prototype = {
 
       if (totalCount === 0 && !this._getInnerText(paragraph, false))
         paragraph.parentNode.removeChild(paragraph);
-    });
+    }, true);
 
-    this._forEachNode(articleContent.getElementsByTagName("br"), function(br) {
+    this._forEachNode(this._getAllNodesWithTag(articleContent, ["br"]), function(br) {
```

Comment:
This removes items as well. Shouldn't this just call `_forEachNode` with the 'backward' param set to `true` and the direct result of getElementsByTagName? That seems more straightforward...